docker/README.md

```diff
@@ -0,0 +1,175 @@
+# Puppet-bolt container
+
```

Comment:
I'm not attached to naming the container `puppet-bolt` it is just what I tagged it when I built with:
```
cas@cas-ThinkPad-T460p:~/working_dir/bolt-vanagon/docker$ docker build . -t  puppet-bolt
```

Reply:
`puppet-bolt` seems good. And just fyi the `Makefile` in the docker dir will let you lint, build, test, and publish